### ISSUE TITLE
Do Not Restrict H-Feed to Only Contain H-Entry

### ISSUE THREAD

**Issue by dshanske** [microformats/h-feed#3]
https://github.com/microformats/h-entry/issues/16

This should actually be an h-feed issue, not an h-entry issue.

**Comment by dshanske** [comment]
Thoughts...

- An h-feed of h-cards could be a contact, follow venue list?
- An h-feed of h-events could be an upcoming or past events list

**Comment by btrem** [comment]
I'm thinking about an opening-hours microformat. Jacky (last name?) suggested a [feed of h-events](https://microformats.org/wiki/opening-hours-brainstorming#Jacky.27s_Idea_for_Rendering_a_Week_of_Hours). I think a new h-feed property called p-feed-type or p-children-type or something like would be useful. Authors could create a feed of h-events or h-cards as @dshanske suggested. And new event type h-opening-hours could be a special type of event. When an h-feed is a collection of h-opening-hours, it's the operating hours of a business.

**Comment by barnabywalters** [comment]
Thinking about this from a user’s perspective: an h-feed represents something which could meaningfully be subscribed to, and/or presented in a feed reader. By default this includes anything with `name`, `content` and `published` properties. The presence of an `author`, either at the feed or child level is also usually implied. Additionally, I can imagine a few other useful cases which fit into the same concept.

* an h-feed containing h-entries makes sense as a feed of posts you could subscribe to in a feed reader
* an h-feed containing h-events makes sense as a feed of events you could subscribe to in a calendar
* an h-feed of h-cards could potentially represent a chronologically ordered list of people. In this case, I’m not so convinced that an h-feed makes more sense than just having a bunch of h-cards on a page, except that wrapping them in an h-feed allows you to name it and assign an author to it. I had a few ideas about how to use things like this in the past, e.g. as a modern blogroll.

**Comment by capjamesg** [comment]
@barnabywalters I agree. I could see myself doing this for a feed of h-reviews and even h-recipes (for subscribing to the ways in which I brew coffee at home). A feed of h-cards could be used as an extended blog roll, potentially, or for similar use cases wherein one links to multiple other personal sites.

**Comment by capjamesg** [comment]
One could take this even further and create a h-feed for a trip using h-geo markup. I am only starting to support h-geo on my blog but I could see a use for a feed that shows, say, all of the h-geo checkins I have for a trip that I have taken.

**Comment by calebhearth** [comment]
I'm using h-feed to contain a heterogenous collection of h-entry and h-cite on https://calebhearth.com/now and my comments (web mentions) sections on various pages including https://calebhearth.com/this-gem-is-mentionable are h-feeds of h-cite.

**Comment by btrem** [comment]
There appears to be no read-only subscribe-able address book format, only CardDAV, which is both overkill and difficult to implement. Consider these Reddit posts:
- https://learn.microsoft.com/en-us/answers/questions/4533257/read-only-contact-group
- https://www.reddit.com/r/selfhosted/comments/89fxwx/readonly_carddavldap_server/
- https://www.reddit.com/r/jamf/comments/1279x0g/readonly_contact_sharing_via_carddavcontact/
- Addressbook sharing as read-only Kozea/Radicale#1458

ISTM that an h-feed of h-cards could provide an alternative to solve the need.

In a similar vein, an h-feed of h-events could provide a simpler read-only alternative to CalDAV which, while less of a nightmare than CardDAV, is still difficult to correctly publish and unnecessary for read-only feeds.

If these formats were adopted, we'd need a clearly defined method of distinguishing an a h-card addressbook or an h-event calendar from a generic feed of different h-types.

**Comment by gRegorLove** [comment]
Merging in comments from earlier issue on the h-entry repo:

From: @jansauer:

> based on the wiki h-feed only allows for nested h-entry objects representing the items of the feed. I would propose removing this. In order to allow any h-* microformat.
> 
> This would allow feeds of events like a event series. That could be used by clients like a calendar software. Or a feed of my friends i am following.
> 
> One example of prior art would be http://tantek.com/ who has already h-event chields inside his h-feed.

From @martymcguire:

> My site's main h-feed at https://martymcgui.re/posts/ contains a mix of h-entry and h-event.
> 
> I also have an h-feed on my homepage for upcoming events which contains only h-event (which is sometimes empty, when I have no upcoming events I'm publicizing).

And some more examples:

* https://unicyclic.com/indieweb/directory h-feed of h-card
* https://gregorlove.com/events/past h-feed of h-event ( and https://gregorlove.com/events/ when I have upcoming events)
* https://indieweb.org/Parse_This WordPress plugin is agnostic about what constitutes a feed
